src/components/SongList/SongList.tsx

```diff
@@ -3,47 +3,29 @@ import { useQuery } from "@apollo/client";
 import { GET_ALL_MUSIC } from "../../graph-ql/queries/GET_ALL_MUSIC/getAllMusic";
 import { GetAllMusic } from "../../graph-ql/queries/GET_ALL_MUSIC/__generated__/GetAllMusic";
 
-// antd imports
-import { List } from "antd";
-
 // types
 import { SongListProps } from "./SongList.types";
 
-// custom-component imports
-import { TitleNode, SongNode } from "./ListItemNodes";
+import SongListItem from "./SongListItem";
 
 const SongList: React.FC<SongListProps> = ({ playSong }) => {
-  const { loading: isLoadingAllMusic, data: allMusicConnection } =
-    useQuery<GetAllMusic>(GET_ALL_MUSIC, {
-      variables: {
-        currentTime: Math.floor(Date.now() / 1000).toString(),
-      },
-    });
+  const { data: allMusicConnection } = useQuery<GetAllMusic>(GET_ALL_MUSIC, {
+    variables: {
+      currentTime: Math.floor(Date.now() / 1000).toString(),
+    },
+  });
 
   return (
     <>
-      {/* songlist */}
-      <List
-        loading={isLoadingAllMusic}
-        style={{
-          width: "700px",
-          alignSelf: "center",
-          border: "1px solid #e5e5e5",
-          borderRadius: "20px",
-          padding: "1em",
-        }}
-        itemLayout="horizontal"
-        dataSource={allMusicConnection?.musicNFTs}
-        renderItem={(item) => (
-          <List.Item>
-            <List.Item.Meta
-              title={<TitleNode musicItem={item} />}
-              description={`Burna Boy`}
-            />
-            <SongNode musicItem={item} playSong={playSong} />
-          </List.Item>
-        )}
-      />
+      <div className="flex flex-wrap justify-center mb-4">
+        {allMusicConnection?.musicNFTs.map((e) => (
```

Comment:
It's always better to be long and clear than short and confusing. Please be taking note of these things. We don't have to make a big deal out of it now but, we'll certainly need to discuss it when we start writing down the conventions and patterns moving forward

Reply:
I don't understand what you said, but if you talking about `e` then any dev experienced with JS can understand it since it is just a map function and I found it unnecessary to name it